README.md

```diff
@@ -16,204 +16,6 @@ This is the repository for Couchbase's Terraform-Provider-Capella which forms a
 ## Using the Provider
 
 ### Building
```

Comment:
can you also link the [CONTRIBUTING.md] here in 1 sentence, so folks can easily navigate to it from the readme? Thanks